api/src/services/entities/transformers/resolveFindOperator.ts

```diff
@@ -0,0 +1,21 @@
+import { FindOperator } from "typeorm"
+
+const resolveFindOperator = <TInput, TOutput>(
```

Comment:
What does this transformer do? Do we need to unit test it to make sure it behaves as expected?

Reply:
This is ported directly from the UI, so we really just treated it as a library.